master/docs/manual/cfg-www.rst

```diff
@@ -571,12 +574,9 @@ Here is a configuration that is known to work (Apache 2.4.10 / Debian 8), direct
         ServerName buildbot.example
         ServerAdmin webmaster@buildbot.example
 
-        <Location /ws>
-          ProxyPass ws://127.0.0.1:8020/ws
-          ProxyPassReverse ws://127.0.0.1:8020/ws
-        </Location>
-
-        ProxyPass /ws !
+        # replace with actual port of your Buildbot master
+        ProxyPass ws://127.0.0.1:8020/ws
```

Comment:
Gosh you're right, seems I made a copy-paste error while contributing to the doc, and my eyes siliently corrected the line each time I reread it (including this morning).

That being said, no the previous version doesn't work on 2.4.25, and that's because of the ProxyPass /ws !

For the record, my configurations on Debian 9 and 8 are managed by configuration manager, using the same template.

Anyway, yes, the /ws is missing and the simplest is to add it, alongside with the info that it also works on CentOS  version.